build.gradle

```diff
@@ -26,30 +26,32 @@ plugins {
   id 'net.researchgate.release' version '2.8.1'
 }
 
+def repositoryName = "EthSigner"
+
 if (!JavaVersion.current().java11Compatible) {
-  throw new GradleException("Java 11 or later is required to build EthSigner.\n" +
+  throw new GradleException("Java 11 or later is required to build " + repositoryName + ".\n" +
   "  Detected version ${JavaVersion.current()}")
 }
 
 def bintrayUser = project.hasProperty('bintrayUser') ? project.property('bintrayUser') : System.getenv('BINTRAY_USER')
 def bintrayKey = project.hasProperty('bintrayApiKey') ? project.property('bintrayApiKey') : System.getenv('BINTRAY_KEY')
 def bintrayPackage = bintray.pkg {
   repo = 'pegasys-repo'
-  name = 'ethsigner'
+  name = repositoryName.toLowerCase()
```

Comment:
Would be nice to not have to do toLowerCase() everywhere, define another variable for this value?

Reply:
done - nicer.